src/Microsoft.AspNet.Mvc.Core/Rendering/StringCollectionTextWriter.cs

```diff
@@ -22,6 +22,15 @@ public class StringCollectionTextWriter : TextWriter
         private readonly Encoding _encoding;
 
         /// <summary>
+        /// Creates a new instance of <see cref="StringCollectionTextWriter"/> with <see cref="Encoding"/>
+        /// defaulting to <see cref="UnicodeEncoding"/>.
+        /// </summary>
+        public StringCollectionTextWriter()
```

Comment:
why add something to the public surface that isn't even used in tests (except tests of the new constructor)?